app.py

```diff
@@ -286,6 +286,22 @@ def get_environment_route(environ):
     else:
         return forbidden_response()
 
+
+@app.route('/environments/{environ}/delete', methods=['DELETE'])
+def delete_environment(environ):
+    """
+    Takes a DELETE request and purges the foursight environment specified by 'environ'.
+    NOTE: This only de-schedules all checks, it does NOT wipe data associated with this
+    environment - that can only be done directly from S3 (for safety reasons).
+
+    Protected route
+    """
+    if check_authorization(app.current_request.to_dict(), environ):
```

Comment:
I looked at the check_authorization function and while it looks probably-OK, I think it should not be maintained as part of foursight. We should centralize all authy things in a repo that is ultimately maintained by devops, and probably supplies decorators that are purely declarative for code so that we know things are being done in a standard way that has been vetted. I don't think that has to be done on this PR, but a # TODO (C4-138): comment would be nice. (That's the ticket number I just created.)